sip-quickstart/.env

```diff
@@ -1,3 +1,14 @@
+# description: The name of your SIP application
+# format: text
+# required: true
 APP_NAME=SIP Quickstart
```

Comment:
pretty sure i just merged code to trim out any whitespace found in the values, so this would end up as `SIPQuickstart` in the deployed code

Reply:
Why do you trim out whitespace? What if a value contains for example the content for a text message?

Reply:
generally i've seen issues with .env file parsers inferring the space to trigger the start of a new commend so it was more trying to get ahead of that, though granted by the time the backend has this data its just a string. I guess i've always viewed spaces in the value as invalid

Reply:
This will definitely cause problems in this specific case, should they be quoted?